shop-domain/src/main/java/com/maxshop/model/category/Category.kt

```diff
@@ -0,0 +1,3 @@
+package com.maxshop.model.category
+
+data class Category(val categoryName: String, val imageUrl: String)
```

Comment:
rename first argument into `name`. It's obvious that it's category name coz this field in class `category`

Reply:
fixed